frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
delete?

Reply:
or add some docs saying that this should be replaced with our own stuff

Reply:
hard to comment in a JSON file (since those don't allow comments). Will create an issue instead for this

Reply:
added issue #1901